experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -175,15 +175,17 @@ describe("TinyliciousClient post-initialization", () => {
                 map1: SharedMap,
             },
         };
-        const [containerCreate] = await ResettableTinyliciousClient.createContainer(containerConfig, schema);
+        const containerCreate = await ResettableTinyliciousClient.createContainer(containerConfig, schema);
```

Comment:
Variable naming here and below of the result of `createContainer` might want to be `containerAndServices`? Or we could directly assign `containerCreate` to the `.fluidContainer` value without creating a new variable

Reply:
The main reason for defining a separate variable was, there would be an error " This line has a length of 124. Maximum allowed is 120 max-len". To reduce the maximum number of characters in the line, I had to define a new variable.

Reply:
Ah gotcha, perhaps we can break that same line apart into separate lines?
```suggestion
        const containerCreate = await ResettableTinyliciousClient.createContainer(containerConfig, schema);
        const containerCreate = (await ResettableTinyliciousClient.createContainer(containerConfig, schema))
           .fluidContainer;
```